api-catalog-ui/frontend/src/components/Header/Header.jsx

```diff
@@ -81,49 +90,79 @@ function Header(props) {
                     </div>
                 </Link>
                 <Link href={dashboard}>
-                    <Typography variant="subtitle2">API Catalog</Typography>
+                    <Typography id="product-title" variant="subtitle2">
+                        API Catalog
+                    </Typography>
                 </Link>
             </div>
             <div className="right-icons">
-                <div className="logout-container">
-                    <Button
-                        className={classes.root}
-                        data-testid="logout-menu"
-                        aria-controls={open ? 'basic-menu' : undefined}
-                        aria-expanded={open ? 'true' : undefined}
-                        aria-haspopup="true"
-                        aria-label="more"
-                        onClick={openMenu}
-                        endIcon={<KeyboardArrowDownIcon id="down-arrow" />}
+                {docLink && (
+                    <Link
+                        data-testid="internal-link"
+                        id="internal-link"
+                        rel="noopener noreferrer"
+                        target="_blank"
+                        href={docLink[1]}
                     >
-                        {s}
-                    </Button>
-                    <StyledMenu
-                        keepMounted
-                        open={open}
-                        onClose={closeMenu}
-                        anchorEl={anchorEl}
-                        getContentAnchorEl={null}
-                        anchorOrigin={{
-                            vertical: 'bottom',
-                            horizontal: 'center',
-                        }}
-                        transformOrigin={{
-                            vertical: 'top',
-                            horizontal: 'center',
-                        }}
-                    >
-                        <div id="profile-menu">
-                            <Typography variant="subtitle2" gutterBottom component="div" id="user-info-text">
-                                Logged in as <strong>{username}</strong>
-                            </Typography>
-                            <Divider />
-                            <MenuItem id="logout-button" data-testid="logout" onClick={handleLogout}>
-                                Log out
-                            </MenuItem>
-                        </div>
-                    </StyledMenu>
-                </div>
+                        {docLink[0]}
+                        <img id="img-internal-link" alt="Internal doc" src={customDoc} />
+                    </Link>
+                )}
+                {process.env.REACT_APP_API_PORTAL === 'true' && (
```

Comment:
I think I submitted another comment, I would move this to some function `isAPIPortal()` if possible

Reply:
done